src/jit/gtstructs.h

```diff
@@ -85,7 +85,9 @@ GTSTRUCT_1(FptrVal     , GT_FTN_ADDR)
 GTSTRUCT_1(Intrinsic   , GT_INTRINSIC) 
 GTSTRUCT_1(Index       , GT_INDEX)
 GTSTRUCT_1(IndexAddr   , GT_INDEX_ADDR)
-#ifdef FEATURE_SIMD
+#if defined(FEATURE_HW_INTRINSICS) && defined(FEATURE_SIMD)
```

Comment:
We covered four cases (Just HWIntrinsic, Just SIMD, both HWIntrinsic and SIMD, and Neither) in the similar `#ifdef` above, we should do the same here.

Reply:
Fixed.